src/views/addProperty/addProperty.jsx

```diff
@@ -30,219 +31,198 @@ const validationSchema = Yup.object().shape({
   managers: Yup.array()
 });
 
-const formHandler = (data, context) => {
-  context.apiCall('post', '/properties', data, { success: 'Property Added!' });
-};
 
-export class AddProperty extends Component {
-  constructor(props) {
-    super(props);
+export const AddProperty = (props) => {
+  const [propertyManagers, setPropertyManagers] = useState([])
+  const [managerOptions, setManagerOptions] = useState([])
+  const [managerSelection, setManagerSelection] = useState([])
+  const [managerSearch, setManagerSearch] = useState('')
+  const userContext = useContext(UserContext)
+  const { showPageTitle, handleCancel } = props
 
-    this.state = {
-      assignedPropertyManagers: [],
-      managerOptions: [],
-      managerSelection: [],
-      managerSearch: ''
-    };
-    this.getManagers = this.getManagers.bind(this);
-  }
-
-  componentDidMount() {
-    this.getManagers(this.context);
-  };
-
-  componentDidUpdate(prevProps, prevState) {
-    if(this.state.managerSearch === prevState.managerSearch) return;
+  useMountEffect(() => getManagers())
 
-    let choices = this.state.managerOptions.filter(
-      manager => manager.description.toLowerCase().includes(this.state.managerSearch.toLowerCase())
-    );
-    this.setState({ managerSelection: choices });
-  }
-
-  getManagers = (context) => {
-    context.apiCall('get', `/user?r=${RoleEnum.PROPERTY_MANAGER}`, {}, {})
+  const getManagers = () => {
+    userContext.apiCall('get', `/user?r=${RoleEnum.PROPERTY_MANAGER}`, {}, {})
       .then((response) => {
         const managerOptions = response.data.users.map(({ id, firstName, lastName }) => {
           return ({
             key: id,
             description: `${firstName} ${lastName}`
-          });
-        });
-        this.setState({ managerOptions, managerSelection: managerOptions });
-      });
-  };
-
-  handleSearchChange = ({ target }) => {
-    let managerSearch = target.value;
-    if(!managerSearch || managerSearch.length === 0) managerSearch = '';
-    this.setState({ managerSearch });
-  };
+          })
+        })
+        setManagerOptions(managerOptions)
```

Comment:
We'll also want to set the "filteredManagerOptions" here so that when the user clicks on the search box but doesn't type anything, they still get a list of entries to choose from